src/renderer/views/Channel/Channel.js

```diff
@@ -249,7 +249,12 @@ export default Vue.extend({
 
     getChannelInfoLocal: function () {
       this.apiUsed = 'local'
+      const id = this.id
```

Comment:
Give it a better name, same for other changes
```suggestion
      const expectedId = this.id
```

Reply:
fixed.